app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import \
+    NotVaccinatedError,\
+    NotWearingMaskError,\
+    OutdatedVaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/210244802-ee064bed-a5df-401a-ad0c-6b0c3301bc86.png)
fix this